.github/workflows/run-tests.yml

```diff
@@ -10,6 +10,7 @@ on:
       - master
       - release-**
       - queued_ltr_backports
+      - 3nids-patch-3
```

Comment:
todo

Reply:
that was to check if you were a serious reviewer ;)